templates/prometheus-instance.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if .Values.prometheusInstance }}
+{{ if and .Values.prometheusInstance .Values.prometheusRemoteWriteRoleArn }}
```

Comment:
Can you put the condition logic in controller so that when arn is not ready, set `prometheusInstance` to false? 

We want to avoid putting too much logic in chart itself.

Reply:
Good call, I'll do that.